### PR TITLE
Add suffixes to typos from "ref" to "rep"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -34643,29 +34643,55 @@ refect->reflect
 refected->reflected
 refecting->reflecting
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34644]: Just noticed lots of preexisting issue here:

1. Actually `refect` is an English verb:
   * [Merriam-Webster](https://www.merriam-webster.com/dictionary/refect)
   
   If you don't mind, could you help moving these to _rare_?
   ```
   refect->reflect
   refected->reflected
   refecting->reflecting
   refects->reflects
   ```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -34643,29 +34643,55 @@ refect->reflect
...
 refectiv->reflective
+refective->reflective
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34646]: 2. Same for `refective`, I would move it to _rare_.
   * [Merriam-Webster](https://www.merriam-webster.com/dictionary/refective)

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -34643,29 +34643,55 @@ refect->reflect
...
+refectivity->reflectivity
 refector->refactor
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34648]: 3. Change from:
   ```
   refector->refactor
   ```
   to:
   ```
   refector->refactor, reflector,
   ```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -34643,29 +34643,55 @@ refect->reflect
...
 refectoring->refactoring
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34649]: 4. Add:
   ```
   refectors->refactors, reflectors,
   ```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -34643,29 +34643,55 @@ refect->reflect
...
 refenrenced->referenced
+refenrences->references
+refenrencing->referencing
 referal->referral
+referals->referrals
 referance->reference
 referanced->referenced
 referances->references
+referancing->referencing
 referant->referent
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34680]: 5. Add plural:
   ```
   referants->referents
   ```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -34643,29 +34643,55 @@ refect->reflect
...
 refere->refer, referee,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:34681]: 6. Add plural / third person:
   ```
   referes->refers, referees,
   ```

Comment (int-y1) [reply codespell_lib/data/dictionary.txt:34681]: `referes->refers, referees,` exists ~50 lines later

**Review by DimitriPapadopoulos — APPROVED** [review]

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**